components/CategoryForm.tsx

```diff
@@ -86,6 +93,17 @@ const CategoryForm = ({
             }
             required
           />
+          <input
+            css={`
+              padding: 6px 24px;
+              width: 80%;
+            `}
+            placeholder="רמז"
```

Comment:
I'd change the placeholder to רמז (לא חובה) so users don't think they have to fill it